docs/getting-started.md

```diff
@@ -0,0 +1,276 @@
+[quickstart guide](quickstart.md) instead.
+
+## Prerequisites
+
+You need to have the following things in order to use firecracker-containerd:
+
+* A computer with a recent-enough version of Linux (4.14+), an Intel x86_64
+  processor (AMD and Arm are on the roadmap, but not yet supported), and KVM
+  enabled.  An i3.metal running Amazon Linux 2 is a good candidate.
+  
+  <details>
+  
+  <summary>Click here to see a bash script that will check if your system meets
+  the basic requirements to run Firecracker.</summary>
+  
+  ```bash
+  err=""; \
+  [ "$(uname) $(uname -m)" = "Linux x86_64" ] \
+    || err="ERROR: your system is not Linux x86_64."; \
+  [ -r /dev/kvm ] && [ -w /dev/kvm ] \
+    || err="$err\nERROR: /dev/kvm is innaccessible."; \
+  (( $(uname -r | cut -d. -f1)*1000 + $(uname -r | cut -d. -f2) >= 4014 )) \
+    || err="$err\nERROR: your kernel version ($(uname -r)) is too old."; \
+  dmesg | grep -i "hypervisor detected" \
+    && echo "WARNING: you are running in a virtual machine. Firecracker is not well tested under nested virtualization."; \
+  [ -z "$err" ] && echo "Your system looks ready for Firecracker!" || echo -e "$err"
+  ```
+  
+  </details>
+* git
+* The Firecracker binary with the optional `vsock` feature enabled.  This
+  feature requires building from source; instructions for doing so are in the
+  [Firecracker getting started guide](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#building-from-source)
+* An uncompressed ELF-format kernel image (you can use the one
+  [described here](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#running-firecracker)
+  as `hello-vmlinux.bin`).
+* A root filesystem image (you can use the one
+  [described here](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#running-firecracker)
+  as `hello-rootfs.ext4`). 
+* A recent installation of
+  [containerd](https://github.com/containerd/containerd/releases).  We suggest
+  [containerd v1.2.1](https://github.com/containerd/containerd/releases/tag/v1.2.1).
+* Go 1.11 or later, which you can download from [here](https://golang.org/dl/).
+
+
+## Setup
+
+### Build Firecracker with `vsock` support
+
+Clone the repository to your computer in a directory of your choice:
+
+```bash
+git clone https://github.com/firecracker-microvm/firecracker.git
+```
+Change into the new directory, and build with Cargo.  Make sure to enable the
+optional `vsock` feature using the `--features vsock` flag.
+
+> Note: Firecracker normally builds a statically-linked binary with musl libc.
+> On Amazon Linux 2, you must specify `--target x86_64-unknown-linux-gnu`
+> because musl libc is not available.  Switching to this target changes the set
+> of syscalls invoked by Firecracker.  If you intend to jail Firecracker using
+> seccomp, you must adjust your seccomp profile for these changes.
+
+```bash
+git checkout v0.12.0 # latest released tag
```

Comment:
[v0.13.0](https://github.com/firecracker-microvm/firecracker/releases/tag/v0.13.0) was released 2 days ago :)

Reply:
I'm going to delay changing this until we've tested it. :smile: